fuzzers/eclipser/fuzzer.py

```diff
@@ -65,7 +65,7 @@ def fuzz(input_corpus, output_corpus, target_binary):
         # Default is low (0.5 sec), recommended to use higher:
         # https://github.com/google/fuzzbench/issues/70#issuecomment-596060572
         '--exectimeout',
-        '2000',
+        '5000',
```

Comment:
Is this change necessary because eclipser breaks otherwise?

Reply:
Yes right, this timeout hits with seed corpus now. Also, it will hit as we add more seed corpus, i remember seeing with libaom that i was planning to add too.